operator/v1/types_console.go

```diff
@@ -39,6 +39,10 @@ type ConsoleCustomization struct {
 	// documentation URL.
 	// Invalid value will prevent a console rollout.
 	DocumentationBaseURL string `json:"documentationBaseURL,omitempty"`
+	//customerProductName is the customizable product name that can be set for the web console
+	CustomProductName string `json:"customProductName"`
+	//customerLogoFile is the name of the file that contains a customized product logo that can be set for the web console
```

Comment:
`logo` seems like a better name based on surrounding customization type.

Reply:
I am matching the name from console

Reply:
> I am matching the name from console

@zherman0 The doc comment doesn't match the JSON value. One says "customer" and one says "custom"

Reply:
fixed